apps/finance/contracts/Finance.sol

```diff
@@ -32,6 +32,25 @@ contract Finance is EtherTokenConstant, IsContract, AragonApp {
     uint256 internal constant NO_PAYMENT = 0;
     uint256 internal constant NO_TRANSACTION = 0;
 
+    string private constant ERROR_COMPLETE_TRANSITION = "FINANCE_COMPLETE_TRANSITION";
+    string private constant ERROR_NO_PAYMENT = "FINANCE_NO_PAYMENT";
+    string private constant ERROR_NO_TRANSACTION = "FINANCE_NO_TRANSACTION";
+    string private constant ERROR_NO_PERIOD = "FINANCE_NO_PERIOD";
+    string private constant ERROR_VAULT_NOT_CONTRACT = "FINANCE_VAULT_NOT_CONTRACT";
+    string private constant ERROR_INIT_PERIOD_TOO_SHORT = "FINANCE_INIT_PERIOD_TOO_SHORT";
+    string private constant ERROR_SET_PERIOD_TOO_SHORT = "FINANCE_SET_PERIOD_TOO_SHORT";
```

Comment:
`ERROR_INIT_PERIOD_TOO_SHORT` and `ERROR_SET_PERIOD_TOO_SHORT` could probably be collapsed into the same error.